reference/features/telemetry.md

```diff
@@ -0,0 +1,76 @@
+# Telemetry
+
+MeiliSearch collects anonymized data from users in order to improve our product. This can be [deactivated at any time](#how-to-disable-data-collection).
+
+Note that while MeiliSearch is running in [`development` mode](/reference/features/configuration.md#environment), no data is collected.
+
+## Why collect telemetry data?
+
+We collect anonymized data for only two reasons: so that we can improve our product, and so that we can continue working on this project full-time.
+
+In order to better address our users' needs and chart a clear path forward for MeiliSearch, we need reliable quantitative information. The information we collect helps us fix bugs, evaluate the success of certain features, and identify our strengths and weaknesses as a product.
+
+Usage metrics also help us justify our existence to investors so that we can keep the MeiliSearch project alive. They prove what we all believe: that MeiliSearch is a great product that can be of use to a large number of people around the world.
```

Comment:
The paragraph on how we use the data we collect is convincing, but the one on the investors feels a bit muddier, as if it's going roundabout instead of saying `we need to prove people are actually using MS`.